scheduler/src/cook/kubernetes/api.clj

```diff
@@ -470,39 +470,103 @@
         ^Callable first-success (->> tmpfn repeatedly (some identity))]
     (.submit kubernetes-executor ^Callable first-success)))
 
+(defn prepare-object-metadata-for-logging
+  "Generates a pruned version of object metadata that is suitable for logging.
+
+  Logging the full metadata object is too much, in particular the
+  managedFields. This function generates a pruned version of metadata
+  that's not as verbose. This cuts logging output by about 2x compared to a
+  simple toString on V1ObjectMeta. As of version 11.0.0 of the Kubernetes
+  client library, the following fields are present in V1ObjectMeta,
+  separated by whether or not we're choosing to log them.
+
+  We do log the following fields because we either use them to determine
+  state in the state machine (e.g. labels) or because we know they are
+  useful for debugging purposes (e.g. resourceVersion):
+
+  - annotations
+  - deletionTimestamp
+  - finalizers
+  - labels
+  - name
+  - namespace
+  - resourceVersion
+
+  We do not log the following fields:
+
+  - clusterName
+  - creationTimestamp
```

Comment:
we're now using prepare-object-metadata-for-logging for different types of metadata, so let's move creationTimestamp from excluded list to included

Reply:
Done